### PR TITLE
Bump istio.io/api

### PR DESCRIPTION
**Pull request by loewenstein**

### CODE REVIEW

**Review by kyessenov — APPROVED** [review]
/lgtm

**Comment by rshriram** [comment]
You just need a sha update right? What are the other changes for ?

File: Gopkg.lock
Comment (ldemailly) [inline Gopkg.lock:1207]: really ? this is going back from alpha2 to alpha1 ?

**Comment by kyessenov** [comment]
Yes, we're going back in time :)

**Comment by loewenstein** [comment]
@rshriram @ldemailly This is just what `make depend.update DEPARGS="--update istio.io/api"` did when I executed it. With only changes in istio.io/api it made sense enough to push the PR.

https://github.com/istio/api/tree/master/authentication/ indeed only contains v1alpha1

**Review by rshriram — APPROVED** [review]

**Comment by ldemailly** [comment]
reminder: please do NOT merge the vendor PR until you're sure you can merge the istio PR just after (ie the istio PR passes all the tests and is approved etc)

**Pull request merged**
